integration/init.go

```diff
@@ -44,6 +51,18 @@ func init() {
 	if clusterID == "" {
 		os.Setenv(EnvVarClusterID, ClusterID())
 	}
+
+	var err error
+	versionBundleVersion, err = framework.GetVersionBundleVersion(service.NewVersionBundles(), TestedVersion())
+	if err != nil {
+		panic(err.Error())
+	}
+	// TODO there should be a not found error returned by the framework in such
+	// cases.
```

Comment:
Why is it todo?

Reply:
Because it should be done in the future in a separate step. I would like to do this but I do not mind when somebody takes this before me.